lib/yapople.js

```diff
@@ -100,9 +100,7 @@ function onData(data) {
         , sData = data.toString()
     ;
     if (typeof this.flow === 'undefined') { // if we have first data chunk from server
-        if (sData.substr(0, 4) === '-ERR') {
-            err = new Error(sData.substring(5, sData.indexOf('\r\n')));
-        } else if (sData.substr(0, 3) === '+OK') {
```

Comment:
Why did you remove the error handling?

Reply:
Most of the time, there was no problem.
In my case, an Unhandled Promise Rejection error occurred on line 104 due to different data formats from the mail server I access sometimes.

Therefore, I changed the code as below.

`
  if (sData.substr(0, 3) === '+OK') {
      succ = sData.substring(4, sData.indexOf('\r\n'));
  } else {
      err = new Error(sData);
  }
`